pkg/e2e/harness_runner/harness_runner.go

```diff
@@ -0,0 +1,38 @@
+package harness_runner
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/onsi/ginkgo/v2"
+	viper "github.com/openshift/osde2e/pkg/common/concurrentviper"
+
+	"github.com/openshift/osde2e/pkg/common/alert"
+	"github.com/openshift/osde2e/pkg/common/config"
+	"github.com/openshift/osde2e/pkg/common/helper"
+	"github.com/openshift/osde2e/pkg/common/prow"
+)
+
+var _ = ginkgo.Describe("[Suite: harnessess] Test Harness", func() {
+	defer ginkgo.GinkgoRecover()
```

Comment:
https://pkg.go.dev/github.com/onsi/ginkgo/v2#GinkgoRecover not sure if this should still be there

Reply:
Removed